From cf9fd2126e697b02e1561501a20dfa13fee16505 Mon Sep 17 00:00:00 2001
From: Armin Kuster <akuster808@gmail.com>
Date: Tue, 18 May 2021 15:10:53 +0000
Subject: [PATCH] sssd: update to 2.5.0

from ../sssd-2.5.0/src/util/sss_pam_data.c:27:
| ../sssd-2.5.0/src/util/debug.h:88:44: error: unknown type name 'uid_t'; did you mean 'uint_t'?
|    88 | int chown_debug_file(const char *filename, uid_t uid, gid_t gid);
|       |                                            ^~~~~
|       |                                            uint_t
| ../sssd-2.5.0/src/util/debug.h:88:55: error: unknown type name 'gid_t'
|    88 | int chown_debug_file(const char *filename, uid_t uid, gid_t gid);
|       |                                                       ^~~~~
| make[2]: *** [Makefile:22529: src/util/libsss_iface_la-sss_pam_data.lo] Error 1
| make[2]: *** Waiting for unfinished jobs....

Upstream-Status: Pending
Signed-off-by: Armin Kuster <akuster808@gmail.com>
---
 src/util/debug.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/util/debug.h b/src/util/debug.h
index c33c14e..405c21d 100644
--- a/src/util/debug.h
+++ b/src/util/debug.h
@@ -24,6 +24,8 @@
 #include "config.h"
 
 #include <stdio.h>
+#include <unistd.h>
+#include <sys/types.h>
 #include <stdbool.h>
 #include <sys/types.h>
 
